pkg/services/provisioning/provisioning.go

```diff
@@ -113,6 +124,12 @@ func (ps *provisioningServiceImpl) ProvisionNotifications() error {
 	return errutil.Wrap("Alert notification provisioning error", err)
 }
 
+func (ps *provisioningServiceImpl) ProvisionUsers() error {
+	usersPath := path.Join(ps.Cfg.ProvisioningPath, "users")
+	err := ps.provisionUsers(usersPath)
+	return errutil.Wrap("Alert notification provisioning error", err)
```

Comment:
```suggestion
	return errutil.Wrap("Users provisioning error", err)
```

Reply:
(same as Sofia's https://github.com/grafana/grafana/pull/24069#discussion_r426614478 -- resolving this and moving suggestion there)